azkaban_cli/azkaban.py

```diff
@@ -540,3 +541,31 @@ def __check_group_permissions(self, permission_options):
             filled_permission_options['read'] = True
 
         return filled_permission_options
+
+    def fetch_flow_execution(self, execution_id):
+        """Fetch a flow execution command, intended to make the request to Azkaban
+        and treat the response properly.
+
+        This method receives the execution id, makes the fetch a flow execution request
+        to fetch the flow execution details and evaluates the response.
+
+        If execution_id is wrong or if there is no session_id, it returns false.
+        If everything is fine, returns True.
```

Comment:
This doesn't look correct

Reply:
You're right. I fixed the docstring now.